.github/workflows/resource-cleanup.yml

```diff
@@ -53,3 +53,76 @@ jobs:
         run: |
           python -m pip install -r requirements.txt
           python cleaner.py
+
+  cleanup-k8s-on-ec2:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        instance: [
+          { repo_name: 'amazon-cloudwatch-agent-operator', ec2_name: 'cw-agent-operator-release', language: 'java' },
+          { repo_name: 'amazon-cloudwatch-agent-operator', ec2_name: 'cw-agent-operator-release', language: 'python' },
+          { repo_name: 'amazon-cloudwatch-agent', ec2_name: 'cw-agent-release', language: 'java' },
+          { repo_name: 'amazon-cloudwatch-agent', ec2_name: 'cw-agent-release', language: 'python' },
+          { repo_name: 'aws-otel-python-instrumentation', ec2_name: 'adot-python-release', language: 'python' },
+          { repo_name: 'aws-otel-java-instrumentation', ec2_name: 'adot-java-release', language: 'java' },
+          { repo_name: 'aws-application-signals-test-framework', ec2_name: 'python-canary', language: 'python' },
+          { repo_name: 'aws-application-signals-test-framework', ec2_name: 'java-canary', language: 'java' } ]
```

Comment:
This is duplicated, is there a way we can have one place for all these so we don't forget them?